numpy/random/mtrand/mtrand.pyx

```diff
@@ -1217,15 +1224,25 @@ cdef class RandomState:
                [3, 2, 2, 0]])
 
         """
-        if high is None:
-            high = low
-            low = 0
-
         key = np.dtype(dtype).name
         if not key in _randint_type:
             raise TypeError('Unsupported dtype "%s" for randint' % key)
```

Comment:
Should this syntax be updated to Python3? using `str.format` should not break in supported versions of Python2.